process/start_project.md

```diff
@@ -69,13 +71,32 @@ Sustainable projects are those the industry adopts and drive economic value with
 
 Academy Software Foundation staff collectively have decades of experience driving open-source projects and ecosystems in various vertical and horizontal industries, bringing the best practices to your project.
 
-- Press relations and analyst relations teams to increase awareness of and excitement about your project, and receive regular reports on press mentions and comparisons to similar projects.
+- Press relations and analyst relations teams to increase awareness of and excitement about your project and receive regular reports on press mentions and comparisons to similar projects.
 - Academy Software Foundation can leverage the expertise of its parent organization, The Linux Foundation, which hosts other successful projects, including the Linux kernel, Let’s Encrypt, and Node.js. 
 - You will have access to full-time Academy Software Foundation staff eager to assist your project in myriad ways and help make it successful.
 
+## What makes a good project to host at the Academy Software Foundation?
+
+Open-source projects and open-collaboration working groups hosted at the Academy Software Foundation are ones that benefit the motion picture and visual effects industries. Successful projects bring diverse organizations and participants together; our experiences at the Linux Foundation and in open source tell us that a diverse community drives greater outcomes and innovation. As a vendor-neutral entity, the Academy Software Foundation creates that level playing field to drive leveraged, open collaboration.
+
+In evaluating projects to be hosted at the Academy Software Foundation, the TAC generally has these considerations when evaluating a project.
+
+- Does the project address a common problem in the motion picture and visual effects industries that are not solved by other efforts?
+- Will the project have the potential to gain alignment and participation amongst multiple constituents, including software vendors, studios, and other related organizations, to participate as contributors and maintainers?
+- Is the project's leadership best positioned and capable of growing the project?
+- If the project grows into an Incubation stage project, could the TAC invest in helping it succeed?
```

Comment:
This talks about "stage 2" of the project lifecycle (Incubation Stage), without mentioning that a project would come in as "Sandbox Stage" (or that a mature project could come in directly at Incubation Stage?).

Reply:
Re-worded, let me know if this is better.